scripts/schema_to_enums_v201.py

```diff
@@ -4,26 +4,29 @@
 import json
 from pathlib import Path
 
-
-def create_class(name):
-    return normalclass(name)
+enum_types = []
+enum_types_names = []
 
 
 def create_attribute(name, casing_style="snake_case"):
```

Comment:
The value of `snake_case` is never used. Wouldn't a boolean more appropriate?

Reply:
I was layering ground for more possible case styles. You prefer me to do that change and make it bool? or include it in a if/elif  and even raise an error in case no answer is provided?

Reply:
Let me know asap

Reply:
> I was layering ground for more possible case styles. 

When do you expect changes to output casing? The casing of Python code in our library is static, either snake_case or CamelCase. I don't see that changing soon.

Reply:
Ok, agreed.